common/src-mapred/org/pentaho/hbase/mapred/PentahoTableInputFormat.java

```diff
@@ -106,6 +106,7 @@ public void configure(JobConf job) {
     
     // set our table record reader
     PentahoTableRecordReader rr = new PentahoTableRecordReader();
+    rr.setHTable( getHTable() );
```

Comment:
Since we are only building cdh54 at present, this change will only be in that shim. However it should be noted that if we build other shims (new or old) from this branch in the future, this change will go into those builds. @lgrill-pentaho FYI